seqr/views/apis/feature_updates_api.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass, asdict
+
+from feedparser import parse
+from markdownify import markdownify as md
+
+import requests
+
+from seqr.views.utils.json_utils import create_json_response
+
+
+@dataclass
+class FeedEntry:
+    """Class representative of one Atom feed entry."""
+
+    author: str
+    author_link: str
+    link: str
+    markdown: str
+    published_datestr: str
+    title: str
+
+
+def get_feature_updates(request):
+    """
+    Fetches the feature-updates GitHub Discussion Atom feed, converts feed entries into markdown, and returns
+    markdown and information for each feed entry.
+    """
+    url = "https://github.com/broadinstitute/seqr/discussions/categories/feature-updates.atom"
+    response = requests.get(url, timeout=5)
+    response.raise_for_status()
+
+    feed = parse(response.content)
+
+    entries = []
+    for entry in feed.entries:
+        # Atom feeds can have multiple content elements per feed entry
+        markdown = "".join(md(content.value) for content in entry.content)
+        entries.append(
+            asdict(
+                FeedEntry(
```

Comment:
Thanks for calling this out. I use dataclasses out of habit, I like creating named classes because I think they're more readable than dictionaries or namedtuples, especially if the object is a parameter in another function. But that's not the case here - the object is only used within the function, and I'm just converting it back to a dictionary. 

I'm going to get rid of the dataclass and just use a dictionary directly

Reply:
^ done